app/templates/auth/change-password.html

```diff
@@ -0,0 +1,122 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Change password – Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+      {
+        "link": "/",
+        "label": "Digital Marketplace"
+      },
+      {
+        "link": dashboard_url,
+        "label": "Your account"
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+{% if form.errors %}
+    <div class="validation-masthead" aria-labelledby="validation-masthead-heading">
+        <h3 class="validation-masthead-heading" id="validation-masthead-heading">
+            There was a problem with the details you gave for:
+        </h3>
+        <ul>
+        {% for field_name, field_errors in form.errors|dictsort|reverse if field_errors %}
+          {% for error in field_errors %}
+            <li><a href="#{{ form[field_name].name }}" class="validation-masthead-link">{{ form[field_name].label.text }}</a></li>
+          {% endfor %}
+        {% endfor %}
+        </ul>
+    </div>
+{% endif %}
+
+<header class="page-heading">
+    <h1>Change your password</h1>
+</header>
+
+<form autocomplete="off" action="{{ url_for('.change_password') }}" method="POST">
+
+    <div class="grid-row">
+        <div class="column-two-thirds">
+            {{ form.hidden_tag() }}
+
+
+            {% if form.old_password.errors %}
```

Comment:
Could we use the toolkit forms stuff for these forms? http://alphagov.github.io/digitalmarketplace-frontend-toolkit/forms/textbox.html

Reply:
It's all copied and pasted from the reset_password template 😸 Happy to do a bit of refactoring so we're consistent across the User FE, but not sure if this branch is the place to do it?

Reply:
Maybe its a tech debt ticket.